src/middle-end/auth/selectors.js

```diff
@@ -22,6 +22,13 @@ exports.getIsAuthenticated = ({ auth }) => {
     return !!(index.result && !index.error);
 };
 
+exports.getHasAuthenticationSettled = ({ auth }) => {
+
+    const { [LOGIN.BASE]: index } = auth.indexes;
+
+    return Boolean(index && (index.result || index.error));
```

Comment:
After successful logout this will return `false` again, since the logout action doesn't return anything (i.e. `index.result` is null/undefined).

Reply:
Oh, that’s that you were trying to tell me in your other repo. What should logout return to address this condition? Just a ‘result’ of ‘true’ or something?

Reply:
I returned `{ loggedOut: true }`, and then had to update the `getIsAuthenticated()` selector to pay attention to that.  The approach gets a little messy handling all these cases, but it works!

Reply:
[I think this takes care of it](https://github.com/BigRoomStudios/strangeluv/pull/242/commits/507b979867f6d76f99f3591af048d60ddff8f62c), but let me know if I'm still missing something ✨

Reply:
That LGTM!